boards/common/remote/include/periph_common.h

```diff
@@ -44,18 +44,22 @@
  */
 static const timer_conf_t timer_config[] = {
     {
+        .num = 0,
```

Comment:
Is this just a coincident or does `num` simply represent the index?

Reply:
yes it does represent the number of the underlying timer device. Originally I omitted this and the implementation assumed the device number by the index in this array. However, that way its not possible to configure timer 2 and 4, only - but one has to use timer 1 and 2. This might be inflexible in certain us cases and or configurations. That's why I (re)introduced the `.num` attribute to allow un-ordered timer initialisation.